charts/crowdsec/values.schema.json

```diff
@@ -99,6 +134,519 @@
+                    "type": "string",
+                    "default": "127.0.0.1",
+                    "description": "Address on which the syslog will listen."
+                },
+                "listen_port": {
+                    "type": "number",
+                    "default": 514,
+                    "description": "UDP port used by the syslog server."
+                },
+                "max_message_len": {
+                    "type": "number",
+                    "default": 2048,
+                    "description": "Maximum length of a syslog message (including priority and facility)."
+                }
+            },
+            "required": [
+                "source"
+            ]
+        },
+        "KafkaSource": {
+            "type": "object",
+            "description": "https://docs.crowdsec.net/docs/next/data_sources/kafka/",
+            "additionalProperties": false,
+            "properties": {
+                "source": {
+                    "const": "kafka"
+                },
+                "labels": {
+                    "$ref": "#/definitions/Labels"
+                },
+                "brokers": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "The name of the kafka brockers to connect to."
+                },
+                "topic": {
+                    "type": "string",
+                    "description": "The topic name you want to read logs from."
+                },
+                "timeout": {
+                    "type": "number"
+                },
+                "tls": {
+                    "type": "object",
+                    "properties": {
+                        "insecure_skip_verify": {
+                            "type": "boolean",
+                            "default": false,
+                            "description": "To disable security checks on the certificate."
+                        },
+                        "client_cert": {
+                            "type": "string",
+                            "description": "Th eclient certificate path. Optional, when you want to enable TLS with client certificate."
+                        },
+                        "client_key": {
+                            "type": "string",
+                            "description": "The client key path. Optional, when you want to enable TLS with client certificate."
+                        },
+                        "ca_cert": {
+                            "type": "string",
+                            "description": "The CA certificate path. Optional, when you want to enable TLS with client certificate."
+                        }
+                    }
+                }
+            },
+            "required": [
+                "source",
+                "brokers"
+            ]
+        },
+        "WinEventLogSource": {
```

Comment:
There is an issue with this validation, if you take a look at the [xpath doc](https://docs.crowdsec.net/docs/next/data_sources/windows_evt_log#xpath_query).

Reply:
Sorry, the schema is good. I had an issue because I had a typo in my values.